### PR TITLE
Module 4.4, Control Flow Practice

### PR DESCRIPTION
**Pull request by pkolodgy**
@bontgoy, requesting a code review. thanks!

### CODE REVIEW

File: ruby/vampires.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			insurance = true
+		else
+			insurance = false
+		end
+
+		# Determine vampirity using conditions
+
+		if name == 'Drake Cula' || name == 'Tu Fang'
+			result = "Definitely a vampire."
+		elsif age != (2016 - birth_year) && !eats_garlic && !insurance
```
Comment (Bontgoy) [inline ruby/vampires.rb]: There is a more dynamic way to work with the year. Google the ruby Time class. You can use commands like `Time.now.year` to make this code scalable in such a way that you would not need to rewrite this file every new year

File: ruby/vampires.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+			result = "Almost certainly a vampire."
+		elsif age != (2016 - birth_year) && ( !eats_garlic || !insurance )
```
Comment (Bontgoy) [inline ruby/vampires.rb]: ^^

File: ruby/vampires.rb [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+			result = "Probably a vampire."
+		elsif age == (2016 - birth_year) && ( eats_garlic || insurance )
```
Comment (Bontgoy) [inline ruby/vampires.rb]: ^^

**Comment by Bontgoy** [comment]
@pkolodgy nice work with control flow and I'm glad to see you already have some comfort with methods.  Check out my suggestions about the `Time` class

**Comment by pkolodgy** [comment]
thanks! using `Time` seems like a great way to handle age validation dynamically. I'll look into it and update the method.

File: ruby/vampires.rb [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+			insurance = true
+		else
+			insurance = false
+		end
+
+		# Determine vampirity using conditions
+
+		if name == 'Drake Cula' || name == 'Tu Fang'
+			result = "Definitely a vampire."
+		elsif age != (Time.now.year - birth_year) && !eats_garlic && !insurance
```
Comment (Bontgoy) [inline ruby/vampires.rb]: gj

**Pull request merged**